### PR TITLE
Fixes crate weigher not going back up when pulled from under crate

### PR DESCRIPTION
**Pull request by SECBATON-GRIFFON**
[bugfix][consistency]

## Changelog
:cl:
 * bugfix: Crate weighers now properly spring up when pulled from under crates.

### CODE REVIEW

File: code/game/cargo_forwarding.dm [hunk 1]
```diff
@@ -246,6 +246,10 @@
 			playsound(get_turf(src), 'sound/effects/spring.ogg', 60, 1)
 			next_sound = world.time + sound_delay
 
+/obj/structure/Uncrossed(atom/movable/A)
+	if(A.type == /obj/machinery/crate_weigher)
+		A.Uncrossed(src)
+
```
Comment (west3436) [inline code/game/cargo_forwarding.dm:260]: ..() needed

**Review by west3436 — CHANGES_REQUESTED** [review]

File: code/game/cargo_forwarding.dm (hunk 1 above)
Comment (SECBATON-GRIFFON) [reply code/game/cargo_forwarding.dm:252]: ah yea sorry, forgot

**Review by jwhitak — CHANGES_REQUESTED** [review]
The smarter fix would be to add this to the crate weigher's Move proc, so that this typecheck isn't called every single time a structure of any sort has something move off of it.

File: code/game/cargo_forwarding.dm (hunk 1 above)
Comment (SECBATON-GRIFFON) [reply code/game/cargo_forwarding.dm:260]: recommended change done instead

**Comment by SECBATON-GRIFFON** [comment]
> The smarter fix would be to add this to the crate weigher's Move proc, so that this typecheck isn't called every single time a structure of any sort has something move off of it.

did this now

**Review by west3436 — APPROVED** [review]

**Review by jwhitak — APPROVED** [review]

**Pull request merged**